doc/Hacking.md

```diff
@@ -1,6 +1,23 @@
 Hacking
 =======
 
+## How to Debug
+
+We have a helper function called `logger.dbg` to help with debugging. You can use that function to print string and tables:
+
+```lua
+local logger = require("logger")
+a = {"1", "2", "3"}
+logger.dbg("table a: ", a)
+```
+
+Anything printed by `logger.dbg` starts with a `#` sign.
```

Comment:
This is the behavior for the deprecated DEBUG function. New logger prints in the following format now:
```
04/06/17-21:44:53 DEBUG foo
```